pkg/qemu-ga/Dockerfile

```diff
@@ -0,0 +1,3 @@
+FROM alpine:3.5
```

Comment:
could you use the new(ish) alpine base image for this? I've spent some time last week to convert all the packages to use it and it would be good to not introduce new package which don't...